src/registry/charts/echarts-radial-chart.tsx

```
@@ -1178,11 +1189,11 @@ export function EChartsRadialChart<TData extends Record<string, unknown>>({
     };
     raf = requestAnimationFrame(tick);
     return () => cancelAnimationFrame(raf);
-  }, [live, isLoading, loadingData]);
+  }, [live, isLoading, loadingData, renderer]);
```

ESLint

[error] Error: Cannot access refs during render

React refs are values that are not needed for rendering. Refs should only be accessed outside of render, such as in event handlers or effects. Accessing a ref value (the `current` property) during render can cause your component not to update as expected (https://react.dev/reference/react/useRef).

/home/jailuser/git/src/registry/charts/echarts-radial-chart.tsx:1192:6
  1190 |     raf = requestAnimationFrame(tick);
  1191 |     return () => cancelAnimationFrame(raf);
> 1192 |   }, [live, isLoading, loadingData, renderer]);
       |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Cannot access ref value during render
  1193 |
  1194 |   // ── Legend overlay (HTML) ─────────────────────────────────────────────────────
  1195 |   // One entry per ring. Unlike the area twin's absolutely-positioned legend, the

(react-hooks/refs)